commit d47395189635098783bfdbb7ccb5ee9f85aa8e37 (refs/changes/94/1094/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-08T16:37:32-08:00 (2 years, 10 months ago)
    
    DOCKER-984 'docker pull some-unreachable-ip/name:tag' takes a LONG time to fail
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/package.json b/package.json
index 255dcac..1e27b36 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -10,7 +10,7 @@
     "bunyan": "1.8.1",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.5",
+    "docker-registry-client": "3.2.6",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
